package.json

```diff
@@ -1,12 +1,12 @@
-{
-  "name": "de.gematik.isik-vitalparameter",
-  "version": "4.0.0-rc",
-  "description": "Package Release des ISiK Modul Vitalparameter und Körpermaße",
-  "fhirVersions": [
-    "4.0.1"
-  ],
-  "dependencies": {
-    "de.basisprofil.r4": "1.5.0-ballot2",
-    "de.gematik.isik-basismodul": "4.0.0-rc2"
-  }
-}
+{
+  "name": "de.gematik.isik-vitalparameter",
+  "version": "4.0.0-rc",
+  "description": "Package Release des ISiK Modul Vitalparameter und Körpermaße",
+  "fhirVersions": [
+    "4.0.1"
+  ],
+  "dependencies": {
+    "de.basisprofil.r4": "1.5.0-ballot3",
```

Comment:
question: ?

Reply:
nötig weil der SCT Code für EKG falsch war in den basisprofilen